tapio/services/rag_orchestrator.py

```
@@ -130,16 +131,30 @@ def query_stream(
 
-            # Step 4: Generate streaming response using LLM service
+            # Step 4: Get the LLM response stream (this calls the LLM service)
             logger.info("Generating streaming response with LLM")
-            response_stream = self.llm_service.generate_response_stream(
+            llm_response_stream = self.llm_service.generate_response_stream(
                 prompt=user_prompt,
                 system_prompt=system_prompt,
             )
 
-            return response_stream, retrieved_docs
+            def stream_generator() -> Generator[str, None, None]:
```

Pylint (3.3.7)

[refactor] Useless return at end of function or method

(R1711)